src/coreclr/scripts/jitutil.py

```diff
@@ -0,0 +1,666 @@
+#!/usr/bin/env python3
```

Comment:
Can you point to the code that was modified that needs reviewed? If we just moved code to this file, then I will skip reviewing it.

Reply:
It's mostly moved code. Some changes:
1. copy_directory now has `verbose_copy` and `verbose_skip` options, to be more granular with the verbosity.
2. some of the functions like `download_one_url` take an `is_azure_storage` argument stating whether the download is coming from Azure blob storage or not (in which case we assume it's a URL or possibly a Windows UNC path).

The biggest thing here is I moved over the download_files function from superpmi.py so I could use its functionality to download a ZIP file, uncompress it, and copy all its contents to a target.